projects/PBOSD.yaml

```diff
@@ -11,7 +11,7 @@ Description: "Probability-based structural design has now emerged as the most ad
 FieldOfScience: Civil Engineering
 ID: '364'
 Name: PBOSD
-Organization: UC San Diego
+Organization: University of California San Diego
```

Comment:
```suggestion
Organization: University of California, San Diego
```

Reply:
From ucsd.edu, it looks like there's no comma in the name.

Reply:
Never mind. They have a comma in their LinkedIn page.